### PR TITLE
simplified heartbeat storage DTO

### PR DESCRIPTION
**Pull request by iulianpascalau**

### CODE REVIEW

File: node/heartbeat/proto/heartbeat.proto [hunk 1]
```diff
@@ -18,21 +16,21 @@ message Heartbeat {
...
+	bool    IsValidator                  = 11;
+	int64	LastUptimeDowntime           = 12;
+	int64   GenesisTime                  = 13;
 }
 
 message DbTimeStamp {
-	google.protobuf.Timestamp TS                    = 1 ;
-}
+	int64 Timestamp = 1;
+}
```
Comment (AdoAdoAdo) [inline node/heartbeat/proto/heartbeat.proto]: newline characters

Comment (iulianpascalau) [reply node/heartbeat/proto/heartbeat.proto]: ok, regarding proxy, will launch a system test and will check. Expected to work accordingly as these structures are use only on local storage.

Comment (sasurobert) [reply node/heartbeat/proto/heartbeat.proto]: add new line to end of file.

Comment (iulianpascalau) [reply node/heartbeat/proto/heartbeat.proto]: done

**Comment by iulianpascalau** [comment]
> check also that proxy is happy

Tested, works as expected.

**Pull request merged**